exporter/exporterparser/jaeger.go

```diff
@@ -0,0 +1,87 @@
+// Copyright 2018, OpenCensus Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package exporterparser
+
+import (
+	"context"
+	"log"
+
+	commonpb "github.com/census-instrumentation/opencensus-proto/gen-go/agent/common/v1"
+	"github.com/census-instrumentation/opencensus-service/exporter"
+	"go.opencensus.io/exporter/jaeger"
+	"go.opencensus.io/trace"
+)
+
+// Slight modified version of go/src/go.opencensus.io/exporter/jaeger/jaeger.go
+type jaegerConfig struct {
+	Endpoint          string `yaml:"endpoint,omitempty"`
+	CollectorEndpoint string `yaml:"collector_endpoint,omitempty"`
+	AgentEndpoint     string `yaml:"agent_endpoint,omitempty"`
```

Comment:
Why 3 endpoints? Wouldn't it just talk to Jaeger collector?

Reply:
I wasn't sure about those but I kept them since they are available on the exporter configuration itself. The exporter code just that handles those are below, with agreement of the parts I certainly can drive the proper changes to any related repo.

https://github.com/census-instrumentation/opencensus-go/blob/master/exporter/jaeger/jaeger.go#L84-L101

Reply:
@pjanotti as @yurishkuro suggested, please just use the `CollectorEndpoint` only since we'll be sending traces to it, those other endpoints are part of the options but unnecessary for this use case.